web_select_all_companies/static/src/xml/switch_all_company_menu.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<templates xml:space="preserve">
+    <t
+        t-name="web_select_all_companies.SwitchAllCompanyMenu"
+        t-inherit="web.SwitchCompanyMenu"
+        t-inherit-mode="extension"
+    >
+        <xpath expr="//t[@t-as='company']" position="before">
+            <div class="dropdown-item d-flex py-0 px-0" data-menu="company">
+                <div
+                    role="menuitemcheckbox"
+                    tabindex="0"
+                    class="ml-auto pl-3 pr-3 border border-top-0 border-left-0 border-bottom-0 toggle_company o_py"
+                >
+                    <span class="height: 2rem;">
+                        <t t-if="widget.isAllCompaniesSelected">
+                            <i class="fa fa-fw fa-check-square pt-2" />
+                        </t>
+                        <t t-else="">
+                            <i class="fa fa-fw fa-square-o pt-2" />
+                        </t>
+                    </span>
+                </div>
+                <div
+                    class="d-flex flex-grow-1 align-items-center py-0 log_into pl-3 o_py"
```

Comment:
Changing this class is an improvement:

```suggestion
                    class="d-flex flex-grow-1 align-items-center py-0 toggle_company pl-3 o_py"
```

Before:
- user has to click the checkbox to toggle all companies
- when user click the text "Select all companies", behavior is confusing

After:
- user can click either the checkbox or the text to select all companies.

Reply:
@florentx  indeed it was causing issue, updated PR - thank you